bench/config/common_site_config.py

```diff
@@ -55,7 +55,7 @@ def get_gunicorn_workers():
 	'''This function will return the maximum workers that can be started depending upon
 	number of cpu's present on the machine'''
 	return {
-		"gunicorn_workers": multiprocessing.cpu_count()
+		"gunicorn_workers": multiprocessing.cpu_count()*2 + 1
```

Comment:
```suggestion
		"gunicorn_workers": multiprocessing.cpu_count() * 2 + 1
```

Fix styling and the commit message to follow semantic messages. refer https://www.conventionalcommits.org/en/v1.0.0/